php-classes/Slate/CBL/Competency.php

```diff
@@ -136,6 +136,16 @@ public function getSkillIds($forceRefresh = false)
         return $skillIds;
     }
 
+    private $totalSkills;
+    public function getTotalSkills($forceRefresh = false)
+    {
+        if (!empty($this->totalSkills)) {
```

Comment:
the difference between `if (!empty($this->totalSkills))` and `if ($this->totalSkills)` is that !empty additionally checks if the variable is defined, and it's impossible for this not to be defined since it's a declared member, so using `empty` here is superfluous